db/structure.sql

```diff
@@ -1282,7 +1282,7 @@ CREATE TABLE users (
     languages character varying(255),
     status user_status_enum DEFAULT 'pending'::user_status_enum NOT NULL,
     terms_agreed timestamp without time zone,
-    consider_pd boolean DEFAULT false NOT NULL,
+    consider_pd boolean DEFAULT true: NOT NULL,
```

Comment:
@jeffreyameyer we shouldn't directly modify this SQL here but perform this change through a migration. I also recommend against changes like these because it gets harder for us to follow the OSM upstream codebase.